app/serializers/node_discovery_serializer.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class NodeDiscoverySerializer < ActiveModel::Serializer
+  include RoutingHelper
+
+  attributes :links
+
+  def links
+    [
+      {
+        rel: 'http://nodeinfo.diaspora.software/ns/schema/2.0',
+        href: node_info_schema_url('2.0'),
+      },
+      {
+        rel: 'http://nodeinfo.diaspora.software/ns/schema/2.1',
```

Comment:
From #10079: there is no `http://nodeinfo.diaspora.software/ns/schema/2.1`. there is no official diaspora nodeinfo 2.1 spec, and the unofficial draft spec should not use a diaspora namespace without consent from diaspora (which i have not seen) 

https://github.com/tootsuite/mastodon/pull/10079#discussion_r258334251

Reply:
Oh, indeed, I haven't thought to check that. I think I'll just remove nodeinfo 2.1.